Fluent/Controls/Backstage.cs

```diff
@@ -378,6 +398,33 @@ private void CreateAndAttachBackstageAdorner()
 
             var layer = AdornerLayer.GetAdornerLayer(this);
             layer.Add(this.adorner);
+
+            layer.CommandBindings.Add(new CommandBinding(RibbonCommands.OpenBackstage,

```

Comment:
However the backstages CommandBindings do not work when the window content is an arbitrary .NET Object. Thus I removed the commandbindings from the backstage and added it to the layer

Von meinem iPhone gesendet

Am 25.05.2015 um 14:17 schrieb Bastian Schmidt <notifications@github.com<mailto:notifications@github.com>>:


In Fluent/Controls/Backstage.cs<https://github.com/fluentribbon/Fluent.Ribbon/pull/118#discussion_r30976996>:

> @@ -378,6 +398,33 @@ private void CreateAndAttachBackstageAdorner()
>
>              var layer = AdornerLayer.GetAdornerLayer(this);
>              layer.Add(this.adorner);
> +
> +            layer.CommandBindings.Add(new CommandBinding(RibbonCommands.OpenBackstage,


This does not seem to be related to your fix. Why are you adding a new CommandBinding to the layer? The Backstage itself already has such a CommanBinding.

-
Reply to this email directly or view it on GitHub<https://github.com/fluentribbon/Fluent.Ribbon/pull/118/files#r30976996>.